koku/api/dataexport/models.py

```diff
@@ -0,0 +1,59 @@
+"""Models for user-initiated data exports."""
+
+from uuid import uuid4
+
+from django.db import models
+
+from api.iam.models import User
+
+
+class DataExportRequest(models.Model):
+    """A user's request for a data export."""
+
+    PENDING = 'pending'
+    COMPLETE = 'complete'
+    STATUS_CHOICES = ((PENDING, 'Pending'), (COMPLETE, 'Complete'))
+
+    uuid = models.UUIDField(default=uuid4, editable=False, unique=True, null=False)
+    created_by = models.ForeignKey('User', null=False, on_delete=models.CASCADE)
```

Comment:
Definitely something that can (if needed) be iterated on in future PRs, but we typically associate things to a customer/provider too so we a) know which tenant to pull the data from and b) which data in the tenant needs pulling. User is obviously useful because we get an email address out of it, but we may end up needing more as this is implemented.

Reply:
Sounds good! I'll hold onto that suggestion for future iterations. 👍